artman/pipelines/grpc_generation.py

```diff
@@ -118,7 +118,7 @@ def get_grpc_codegen_tasks(self, **kwargs):
             protoc_tasks.ProtoDescGenTask,
             protoc_tasks.ProtoAndGrpcCodeGenTask,
             package_metadata_tasks.PackageMetadataConfigGenTask,
-            package_metadata_tasks.PythonProtoPackageMetadataGenTask,
```

Comment:
Please make sure you test this. The original `ProtoPackageMetadataGenTask` change broke my grpc packages (they were copied with no metadata).